php/admin-functions-term.php

```diff
@@ -0,0 +1,105 @@
+<?php
```

Comment:
@kjbenk In the functions with `_add_term_` we don't have a `$term_id` just due to the screen.